cluster-provision/k8s-multus/scripts/provision.sh

```diff
@@ -50,24 +57,32 @@ yum install --nogpgcheck -y \
     kubelet-${version} \
     kubectl-${version} \
     kubernetes-cni-0.6.0 \
-    openvswitch
+    http://cbs.centos.org/kojifiles/packages/openvswitch/2.10.1/3.el7/x86_64/openvswitch-2.10.1-3.el7.x86_64.rpm
```

Comment:
why do use specific version?

Reply:
they remove the openvswitch from the centos repo and also from the epel-release so here I just point it to a specific version. (same one that it used by openshift)